.github/workflows/fuzz.yml

```
@@ -13,16 +13,25 @@ jobs:
       run: |
         set -e
         rustup set profile minimal
-        rustup toolchain install nightly 
+        rustup toolchain install nightly
         rustup default nightly
     - uses: taiki-e/install-action@v2
-      with: 
+      with:
         tool: cargo-bolero
     - run: |
         set -e
-        (cd alloc && cargo bolero test chain::tests::fuzz -T 1min)
-        (cd alloc && cargo bolero test linear::tests::fuzz -T 1min)
-        (cd profiling && cargo bolero test collections::string_table::tests::fuzz_arena_allocator -T 1min)
-        (cd profiling && cargo bolero test collections::string_table::tests::fuzz_string_table -T 1min)
-
-
+        DIRS="alloc profiling"
+        for dir in $DIRS;
+        do
+          # cargo bolero list outputs {"package":"package-name","test":"test-name"}
+          (cd $dir && cargo bolero list | \
+          # And the following command will parse package-name's and test-name's one in each line
+          grep -oP '"(package|test)"\s*:\s*"\K[^"]+' | \
+          # awk will stitch package and test names back separated by a tab
+          awk 'NR%2{printf "%s\t", $0; next}1' | \
+          while read -r package test;
+          do
+            echo "Starting bolero test for $package $test"
```

Might be easier to scan for if you put a pattern here e.g.
```suggestion
            echo "****** Starting bolero test for $package $test ******"
```